components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -672,16 +680,10 @@ void FeedV2Builder::BuildPublisherFeed(const std::string& publisher_id,
                 continue;
               }
 
-              result->items.push_back(
-                  mojom::FeedItemV2::NewArticle(item->get_article()->Clone()));
+              items.push_back(item->Clone());
             }
 
-            base::ranges::sort(result->items, [](const auto& a, const auto& b) {
```

Comment:
Just double checking, we don't need sorting based on time and pop recency any more?

Reply:
I had a chat with lorenzo about it and I think it's fine. Channels should use exactly the same strategy for sorting as the main feed, and we're not sure what to do with publishers yet. In some cases, just sorting by recency makes the most sense, but in others, pop_score is more handy.